src/components/List.js

```diff
@@ -183,42 +183,37 @@ const List = ({ token }) => {
                         item.itemName
                       } ready to purchase ${backgroundColor(item)}`}
                     >
-                      <Checkbox
-                        type="checkbox"
-                        icon={
-                          <CircleUnchecked
-                            style={{
-                              color: checkboxColor[backgroundColor(item)],
-                            }}
+                      <div className="flex-container">
+                        <div>
```

Comment:
This approach of using an extra div and the justify-content works OK 👍 
I want to point out that you could also avoid using this extra div, and avoid the stretching of the delete button we're seeing, by using some other flex properties instead.
You could delete this div, and then replace your css with this:
```
.flex-container {
  display: flex;
  align-items: center; // you need this to vertically center the text
  margin-top: 1rem;
}

.flex-container button {
  margin-left: auto; // this "floats" the delete button to the right hand side
}
```

![delet button](https://user-images.githubusercontent.com/2433616/109097047-7b13bc00-76d3-11eb-9383-8d94914ebbfd.PNG)

Reply:
Cool, thanks for the tip on that. We may need that later on for other buttons! We'll be replacing the delete button on the list view with a trash can icon soon 🗑️